README.md

```diff
@@ -144,3 +144,25 @@ client.autocomplete({
   console.log(e);
 });
 ```
+
+## Business Match
+```javascript
+'use strict';
+
+const yelp = require('yelp-fusion');
+
+const client = yelp.client(token);
+
+client.autocomplete('lookup', {
```

Comment:
Typo. Should be ```client.businessMatch(...)```

Reply:
Can you also add a comment here indicating valid values for matchType are 'lookup' and 'best'